### DISCUSSION TITLE
Where's my hard drive? - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#41] [General]
# Where's my hard drive? - Alan Pope's blog

This is the fourth in a series of Friday Tales From Tech Support. Some stories from the past featuring broken computers and even more broken tech support operatives - mostly me.
Scene setting Today’s story is another belter from my stint on the helpdesk for a large accounting & consulting organisation in central London. It’s around 1995 and I’m a happy-go-lucky, young, free and single tech support operative. Always happy to help, and generally enjoying the work.

https://popey.com/blog/2023/09/wheres-my-hard-drive/

**Comment by Unixware** [comment]
Similar cr*p happened to me ALL of the effing time that era, (l)users lost their files because they had no clue where they put them, needless to say I enjoyed it (evil grin)
